app/routes/codelijsten.js

```diff
@@ -1,3 +1,9 @@
 import Route from '@ember/routing/route';
 
-export default class CodelijstenRoute extends Route {}
+export default class CodelijstenRoute extends Route {
+  setupController(controller, model) {
+    super.setupController(...arguments);
+
+    controller.setup(model);
```

Comment:
avoid using the setupController to act as an init lifecyclehook, better place to do this is the constructor of the controller itself. also specifcally in this case, the setup function does not expect any arguments

Reply:
The constructor is only called initially what means when leaving the `codelijsten` route and than coming back it is not called unfortunately. Also the [glimmer component](https://guides.emberjs.com/release/upgrading/current-edition/glimmer-components/#toc_lifecycle-and-properties) only has two lifecycle hooks to use.. Fixed this by calling the `@action updateSelectedSchemeTo` in the template so it is called everytime the template loads.